src/nxt_http.h

```diff
@@ -9,6 +9,9 @@
 
 #include <nxt_regex.h>
 
+#if (NXT_HAVE_OTEL)
+#include <nxt_otel.h>
+#endif
```

Comment:
then do you want the nxt_otel.h content to be in a similar conditional?

Reply:
> This change was only made because you asked for it in the last review.

But then I realised I had it backwards and said [this](https://github.com/nginx/unit/pull/1431#discussion_r1764227085)

> Then do you want the nxt_otel.h content to be in a similar conditional?

Not the whole thing...  you may need to wrap some parts of it in conditionals.

Look at [src/nxt_tls.h](https://github.com/nginx/unit/blob/master/src/nxt_tls.h) or [this](https://github.com/ac000/unit/blob/compr/src/nxt_http_compression.h) for examples.

Reply:
In fact it's all good, no need for any #ifdeferry 

You can simply do this

```diff
diff --git ./src/nxt_http_error.c ./src/nxt_http_error.c
index be5aef10..4cafb872 100644
--- ./src/nxt_http_error.c
+++ ./src/nxt_http_error.c
@@ -6,11 +6,7 @@
 
 #include <nxt_router.h>
 #include <nxt_http.h>
-
-
-#if (NXT_HAVE_OTEL)
 #include <nxt_otel.h>
-#endif
 
 
 static void nxt_http_request_send_error_body(nxt_task_t *task, void *r,
diff --git ./src/nxt_http_request.c ./src/nxt_http_request.c
index 4a208b58..1c3487ae 100644
--- ./src/nxt_http_request.c
+++ ./src/nxt_http_request.c
@@ -6,10 +6,7 @@
 
 #include <nxt_router.h>
 #include <nxt_http.h>
-
-#if(NXT_HAVE_OTEL)
 #include <nxt_otel.h>
-#endif
 
 
 static nxt_int_t nxt_http_validate_host(nxt_str_t *host, nxt_mp_t *mp);
```